docs/supported_formats/ripple.rst

```diff
@@ -19,24 +19,19 @@ It can also be used to exchange data with Bruker and used in
 combination with the :ref:`import-rpl` it is very useful for exporting data
 to Gatan's Digital Micrograph.
 
-The default encoding is latin-1. It is possible to set a different encoding
-using the encoding argument, e.g. when using HyperSpy to save a file:
-
-.. code-block:: python
-
-    >>> s.save('file.rpl', encoding = 'utf8')
-
-For ``.rpl`` files, ``hs.load`` takes the ``mmap_mode`` keyword argument enabling
-loading the file using a different mode (default is copy-on-write). However,
-note that lazy loading does not support in-place writing (i.e lazy loading and
-the "r+" mode are incompatible).
-
 .. note::
 
     This format may not provide information on the calibration.
     If so, you should add that after loading the file.
 
 
+API functions
+^^^^^^^^^^^^^
+
+.. automodule:: rsciio.ripple
+   :members:
+
+
 Format description
```

Comment:
Shall this section be moved to the `File Specifications` section of the documentation with a link in the introduction section.

Reply:
I created a dedicated page under `File specifications`.